Makefile

```diff
@@ -1,7 +1,11 @@
+ifdef GREP
+	GREPARG = -g $(GREP)
+endif
+
 REPORTER ?= spec
-UNIT_TESTS = $(shell find ./test/unit -name "test_*.js")
-QPID_INTEGRATION_TESTS = $(shell find ./test/integration/qpid -name "*.test.js")
-SERVICEBUS_INTEGRATION_TESTS = $(shell find ./test/integration/servicebus -name "*.test.js")
+UNIT_TESTS = ./test/unit
+QPID_INTEGRATION_TESTS = ./test/integration/qpid
```

Comment:
hm technically the idea with the `$(shell find..)` was that we wouldn't have to specify explicit paths, but rather just pick up anything below a top level dir. tbh if it works, I don't mind for now :smile: